BUILDING.md

```diff
@@ -15,6 +15,15 @@ or MonoDevelop.
 
 The solutions all place their output in a common bin directory under the solution root.
 
+## Unit Testing
+
+The unit tests that should be run in the solution are grouped by project name:
+ * `nunit.framework.tests-*`
+ * `nunitlite.tests-*`
+ * `slow-nunit-tests-*`
+
+Other test projects contain tests designed to fail purposefully for integration purposes.
```

Comment:
As in "Other test projects contain tests designed to fail purposely for integration tests." ?